tests/framework/clients/test_client.go

```diff
@@ -44,30 +40,18 @@ const (
 )
 
 //CreateTestClient creates new instance of test client for a platform
-func CreateTestClient(platform enums.RookPlatformType, k8sHelper *utils.K8sHelper, namespace string) (*TestClient, error) {
+func CreateTestClient(k8sHelper *utils.K8sHelper, namespace string) (*TestClient, error) {
 	var blockClient contracts.BlockOperator
 	var fsClient contracts.FileSystemOperator
 	var objectClient contracts.ObjectOperator
 	var poolClient contracts.PoolOperator
-	rookRestClient := CreateRestAPIClient(platform, k8sHelper, namespace)
-
-	switch {
-	case platform == enums.Kubernetes:
-		blockClient = CreateK8BlockOperation(k8sHelper, rookRestClient)
-		fsClient = CreateK8sFilesystemOperation(k8sHelper, rookRestClient)
-		objectClient = CreateObjectOperation(rookRestClient)
-		poolClient = CreatePoolClient(rookRestClient)
-	case platform == enums.StandAlone:
-		blockClient = nil  //TODO- Not yet implemented
-		fsClient = nil     //TODO- Not yet implemented
-		objectClient = nil //TODO- Not yet implemented
-		poolClient = nil   //TODO- Not yet implemented
-	default:
-		return &TestClient{}, fmt.Errorf("Unsupported Rook Platform Type")
-	}
+	rookRestClient := CreateRestAPIClient(k8sHelper, namespace)
```

Comment:
why do we have all these different clients? Looks like they have similar properties with k8s and rest client objects.